src/core/TrustManager.sol

```diff
@@ -40,8 +40,8 @@ abstract contract TrustManager is IRegistry {
         attesters.sort();
         attesters.uniquifySorted();
 
-        // if attesters array has duplicates, revert
-        if (attestersLength == 0) revert InvalidTrustedAttesterInput();
+        // if attesters array has duplicates or is too large revert
+        if (attestersLength == 0 || attestersLength > type(uint8).max) revert InvalidTrustedAttesterInput();
```

Comment:
NOTE: adding check to prevent attester length to be over uint8 thus preventing cast issue